mongo.py

```diff
@@ -29,8 +29,8 @@ def remove(self):
             return resp
 
 class Listing(Model):
-    db_client = pymongo.MongoClient('localhost', 27017)
-    collection = db_client["listings"]["listings_list"]
+    client = pymongo.MongoClient("mongodb+srv://<user>:<password>@cluster0.sm94a.mongodb.net/BobsListDB?retryWrites=true&w=majority")
```

Comment:
Very necessary change! The prior version was based on having a local repo, after the change to using Palo alto. This allows for a common repository amongst the people who are working on the project. Simulating how it would work in the real world.

Reply:
Thanks